ui/src/app/layout/topics/deprecation/deprecation.component.ts

```diff
@@ -75,17 +75,13 @@ export class DeprecationComponent implements OnInit {
     }
 
     private toPeriodText(period: { years: number; months: number; days: number }): string {
-        const target = DateTime.now().plus({ years: period.years, month: period.months, days: period.days });
-        // const oldThreshold = DateTime.relativeTimeThreshold('d') as number;
-
-        // special treatment: If days set, avoid moment "rounding" to months
-        // Note: this still produces wrong results for some values of "days"
-        // moment.js should be replaced with a better library.
-        if (period.days) {
-            DateTime.relativeTimeThreshold('d', 99999);
-        }
-        // TODO needs to be cleared what the luxon alternative is for set/get threhold
-        //DateTime.relativeTimeThreshold('d', oldThreshold);
-        return target.toRelativeCalendar({ locale: this.translateService.currentLang });
+        Object.keys(period).forEach(key => {
```

Comment:
Perhaps add a small comment, because this is not self-explanatory:
`// delete zero-valued keys from object to avoid a text like "0 years, 3 months, 0 days"`